src/main/java/domain/AttendanceRecord.java

```diff
@@ -0,0 +1,80 @@
+package domain;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Objects;
+
+public class AttendanceRecord {
+    private static final int DATE_INDEX = 0;
+    private static final int TIME_INDEX = 1;
+    public static final LocalTime ABSENT_TIME = LocalTime.of(14, 0);
+
+    private final LocalDate date;
+    private final LocalTime time;
+    private final Attendance attendance;
+
+    private AttendanceRecord(LocalDate date, LocalTime time) {
+        this.date = date;
+        validateDate(date);
+        this.time = time;
+        this.attendance = Attendance.getAttendanceStatus(Day.getDay(date), time);
+    }
+
+    public static AttendanceRecord parse(String dateTime) {
+        return new AttendanceRecord(LocalDate.parse(dateTime.split(" ")[DATE_INDEX]),
+                LocalTime.parse(dateTime.split(" ")[TIME_INDEX]));
+    }
+
+    public static AttendanceRecord of(LocalDate date, LocalTime time) {
+        return new AttendanceRecord(date, time);
+    }
+
+    public static AttendanceRecord checkIn(LocalTime time, DateGenerator dateGenerator) {
+        return new AttendanceRecord(dateGenerator.generate(), time);
+    }
+
+    public static AttendanceRecord asAbsent(LocalDate date) {
+        return new AttendanceRecord(date, ABSENT_TIME);
+    }
+
+    public LocalDate getDate() {
+        return date;
+    }
+
+    public LocalTime getTime() {
+        return time;
+    }
+
+    public Attendance getAttendance() {
+        return attendance;
+    }
+
+    public boolean isTardy() {
+        return attendance.equals(Attendance.TARDY);
```

Comment:
enum 은 보통 `equals` 보다는 `==` 비교를 사용합니다.

```suggestion
        return attendance == Attendance.TARDY;
```

Reply:
앗 이 부분을 놓쳤네요 감사합니다!

Reply:
4a79a58